pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -196,6 +197,11 @@ func (k *KogitoServiceSpec) SetResources(resources corev1.ResourceRequirements)
 // GetHTTPPort ...
 func (k *KogitoServiceSpec) GetHTTPPort() int32 { return k.HTTPPort }
 
+// SetHTTPPort ...
+func (k *KogitoServiceSpec) SetHTTPPort(httpPort int32) {
+	k.HTTPPort = httpPort
+}
```

Comment:
Sorry to nitpick, but could you please keep this in one line?

Reply:
Hi @ricardozanini ! Sure. Can you share the configuration of VS.Code so we all share the same settings? At the moment, my formatter is making this code in multi-lines...